lib/catalog_inventory/payload.rb

```diff
@@ -1,11 +1,13 @@
 module CatalogInventory
   class Payload
-    attr_accessor :response_format, :upload_url, :jobs
+    attr_accessor :response_format, :upload_url, :jobs, :previous_sha, :previous_size
 
-    def initialize(response_format, upload_url, jobs)
+    def initialize(response_format, upload_url, jobs, previous_sha = nil, previous_size = nil)
       @response_format = response_format
       @jobs = jobs
       @upload_url = upload_url
+      @previous_sha = previous_sha
```

Comment:
@hsong-rh Should this be checking if the value is nil.

Reply:
Check is added